### DISCUSSION TITLE
Inference on Non-PYNQ or Non-Alveo boards

### DISCUSSION THREAD

**Discussion by anguyen9630** [Xilinx/finn#758] [Q&A]
Hey all,

I've got access to a 7-series FPGA and have managed to build a Stitched IP for it. But how do one actually do inference on it? Since it have AXI interface I would image I should use Microblaze and use PyTorch C++ API but I realised that the input have to be ONNX compatible......

PYNQ example I am not sure will help but I haven't managed to find any example for the Alveo to kinda base the implementation on...

Thanks for your help!

**Comment by dj-park** [comment]
It would be really helpful if there's a tutorial on using FINN-generated xclbin files with regular Vitis flow.
I think the main difficulty is on manipulating input data in the host code based on the folding configuration.... it would be useful if we have a tutorial on this.
